Sources/Rendering/Core/Camera/index.js

```diff
@@ -380,7 +380,7 @@ function vtkCamera(publicAPI, model) {
     let t = thickness;
     if (t < 1e-20) {
       t = 1e-20;
-      vtkDebugMacro('Thickness is set to minimum.');
+      vtkDebugMacro('Thi./Sources/Rendering/Core/Camera/index.jsckness is set to minimum.');
```

Comment:
looks odd to me

Reply:
Good catch

Reply:
I don't have access to my dev computer (at the airport now), is it something you could help address ? If not, I will take care of it when I return on March 2nd.